pkg/agent/kfstorage/provider.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package storage
+package kfstorage
```

Comment:
is there a reason for package rename?

Reply:
I was getting conflicts with the `cloud.google.com/go/storage` import, since storage was resolving to the package instead of the gcs library. However, I have now changed the gcs import to have a unique name, so the library will be resolved without changing the package name.

Reply:
you can do: `import gstorage cloud.google.com/go/storage` which functions in go similar to: 
`import cloud.google.com/go/storage as gstorage`